src/corefx/System.Globalization.Native/configure.cmake

```diff
@@ -10,6 +10,9 @@ CHECK_CXX_SOURCE_COMPILES("
 
 if(NOT CLR_CMAKE_PLATFORM_DARWIN)
     set(CMAKE_REQUIRED_LIBRARIES ${ICUUC} ${ICUI18N})
+    if(CLR_CMAKE_PLATFORM_UNIX_ARM)
+        set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES} icudata m stdc++ dl)
```

Comment:
Not all but with gcc-5.3(gcc linker is used) it fails to link as described in https://github.com/dotnet/coreclr/issues/5324#issuecomment-222857590 . I hope there can be better way than check for ARM and add libs.